Sources/Sentry/SentryTracer.m

```diff
@@ -92,6 +98,12 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
             initSlowFrames = currentFrames.slow;
             initFrozenFrames = currentFrames.frozen;
         }
+#endif
```

Comment:
`m`: Can we add some tests for the changes in here? If you need help, just let me know.